SmartApi/smartWebSocketV2.py

```diff
@@ -338,18 +342,32 @@ def send_heart_beat(self):
             raise e
 
     def _on_error(self, wsapp, error):
-        # self.HB_THREAD_FLAG = False
         self.RESUBSCRIBE_FLAG = True
         if self.current_retry_attempt < self.MAX_RETRY_ATTEMPT:
-            print("Attempting to resubscribe/reconnect...")
+            print(f"Attempting to resubscribe/reconnect (Attempt {self.current_retry_attempt + 1})...")
             self.current_retry_attempt += 1
+            if self.retry_strategy == "simple":
```

Comment:
remove simple and exponential from string comparison and use 0, 1 instead . It will be easy to clients.

Reply:
yes removed the simple and exponential from string comparison and use 0, 1